test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -68,8 +68,8 @@ const EventEmitter = require('events');
 }
 
 {
-  const listen1 = function listen1() {};
-  const listen2 = function listen2() {};
+  const listen1 = () => {};
```

Comment:
I just simply would not use `common.noop` here. In fact, if memory serves correctly, when I opened the `common.noop` PR just a few weeks ago I intentionally left this test alone because it did not make sense to use it here... and that is perfectly fine. It's ok *not* to use it when it doesn't make sense to.